client/my-sites/themes/sheet.jsx

```diff
@@ -45,19 +43,7 @@ export const ThemeSheet = React.createClass( {
 	},
 
 	onPrimaryClick() {
-		let action;
-
-		if ( ThemeHelpers.isPremium( this.props ) && ! this.props.purchased && this.props.isLoggedIn ) { //FIXME: purchased ENOENT
-			action = purchase( this.props, this.props.selectedSite, 'showcase-sheet' );
-		} else if ( this.props.active ) { //FIXME: active ENOENT
-			action = customize( this.props, this.props.selectedSite );
-		} else if ( this.props.isLoggedIn ) {
-			action = activate( this.props, this.props.selectedSite, 'showcase-sheet' );
-		} else {
-			action = signup( this.props );
-		}
-
-		this.props.dispatch( action );
+		this.props.dispatch( signup( this.props ) );
```

Comment:
>Was the previous if block a bad copy, or prep for future usage?

It's my assumption, one which should be confirmed by the themes folks, that the other actions were never reached, due to the fact that `selectedSite` was never being passed (current use-cases render the non-connected component). It's also not as simple as replacing with `sitesList.getSelectedSite()`, since the `sites-list` module is not server-render compatible.